# HG changeset patch
# User Rik <rik@octave.org>
# Date 1378308025 25200
#      Wed Sep 04 08:20:25 2013 -0700
# Node ID 3e95b22f5287f0e868de68eca686d8bf15f612a1
# Parent  ddc85ecc901f2232488092355aa1d4e3b7c56d34
doc: Remove private functions from signal documentation chapter (bug #39939).

* doc/interpreter/signal.txi: Remove @DOCSTRING macros for triangle_sw,
triangle_lw, rectangle_sw, rectangle_lw.  Re-order @DOCSTRING macros
to place all fft/ifft functions together.  Remove excess newlines.

diff --git a/doc/interpreter/signal.txi b/doc/interpreter/signal.txi
--- a/doc/interpreter/signal.txi
+++ b/doc/interpreter/signal.txi
@@ -14,84 +14,81 @@
 @c 
 @c You should have received a copy of the GNU General Public License
 @c along with Octave; see the file COPYING.  If not, see
 @c <http://www.gnu.org/licenses/>.
 
 @node Signal Processing
 @chapter Signal Processing
 
-
 This chapter describes the signal processing and fast Fourier
 transform functions available in Octave.  Fast Fourier transforms are
 computed with the @sc{fftw} or @sc{fftpack} libraries depending on how
 Octave is built.
- 
-
-
-@DOCSTRING(detrend)
 
 @DOCSTRING(fft)
 
+@DOCSTRING(ifft)
+
+@DOCSTRING(fft2)
+
+@DOCSTRING(ifft2)
+
+@DOCSTRING(fftn)
+
+@DOCSTRING(ifftn)
+
 Octave uses the @sc{fftw} libraries to perform FFT computations.  When Octave
 starts up and initializes the @sc{fftw} libraries, they read a system wide
 file (on a Unix system, it is typically @file{/etc/fftw/wisdom}) that
 contains information useful to speed up FFT computations.  This
 information is called the @emph{wisdom}.  The system-wide file allows
 wisdom to be shared between all applications using the @sc{fftw} libraries.
 
 Use the @code{fftw} function to generate and save wisdom.  Using the
 utilities provided together with the @sc{fftw} libraries
 (@command{fftw-wisdom} on Unix systems), you can even add wisdom
 generated by Octave to the system-wide wisdom file.
 
 @DOCSTRING(fftw)
 
-@DOCSTRING(ifft)
-
-@DOCSTRING(fft2)
-
-@DOCSTRING(ifft2)
-
-@DOCSTRING(fftn)
-
-@DOCSTRING(ifftn)
-
 @DOCSTRING(fftconv)
 
 @DOCSTRING(fftfilt)
 
 @DOCSTRING(filter)
 
 @DOCSTRING(filter2)
 
 @DOCSTRING(freqz)
 
 @DOCSTRING(freqz_plot)
 
 @DOCSTRING(sinc)
 
 @DOCSTRING(unwrap)
 
-@c FIXME -- someone needs to organize these...
+@c FIXME: someone needs to organize these ...
 
 @DOCSTRING(arch_fit)
 
 @DOCSTRING(arch_rnd)
 
 @DOCSTRING(arch_test)
 
 @DOCSTRING(arma_rnd)
 
 @DOCSTRING(autoreg_matrix)
 
 @DOCSTRING(bartlett)
 
 @DOCSTRING(blackman)
 
+@DOCSTRING(detrend)
+
 @DOCSTRING(diffpara)
 
 @DOCSTRING(durbinlevinson)
 
 @DOCSTRING(fftshift)
 
 @DOCSTRING(ifftshift)
 
@@ -102,31 +99,24 @@ generated by Octave to the system-wide w
 @DOCSTRING(hanning)
 
 @DOCSTRING(hurst)
 
 @DOCSTRING(pchip)
 
 @DOCSTRING(periodogram)
 
-@DOCSTRING(rectangle_lw)
-
-@DOCSTRING(rectangle_sw)
-
 @DOCSTRING(sinetone)
 
 @DOCSTRING(sinewave)
 
 @DOCSTRING(spectral_adf)
 
 @DOCSTRING(spectral_xdf)
 
 @DOCSTRING(spencer)
 
 @DOCSTRING(stft)
 
 @DOCSTRING(synthesis)
 
-@DOCSTRING(triangle_lw)
+@DOCSTRING(yulewalker)
 
-@DOCSTRING(triangle_sw)
-
-@DOCSTRING(yulewalker)
